src/solc_0.8/Game/GameMinter.sol

```diff
@@ -5,10 +5,9 @@ pragma solidity 0.8.2;
 import "./GameToken.sol";
 import "../common/interfaces/IGameMinter.sol";
 import "@openzeppelin/contracts-0.8/token/ERC20/IERC20.sol";
+import "../common/Libraries/ERC2771Handler.sol";
```

Comment:
Did not notice before but ERC2771Handler is not a library.
I would instead add it to the base folder

Reply:
yes, good catch.Will do.